lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -74,21 +118,21 @@ function createPsuedoLocaleStrings(strings) {
       }
     }
 
-    psuedoLocalizedStrings[key] = psuedoLocalizedString.join('');
+    psuedoLocalizedStrings[key] = {message: psuedoLocalizedString.join('')};
   }
 
   return psuedoLocalizedStrings;
 }
 
 /**
  * @param {LH.Locale} locale
- * @param {Record<string, string>} strings
+ * @param {Record<string, ICUMessage>} strings
  */
 function writeStringsToLocaleFormat(locale, strings) {
   const fullPath = path.join(LH_ROOT, `lighthouse-core/lib/locales/${locale}.json`);
   const output = {};
-  for (const [key, message] of Object.entries(strings)) {
-    output[key] = {message};
+  for (const [key, object] of Object.entries(strings)) {
```

Comment:
nit: above `defn` is used...might be better than `object` here

Reply:
done